### DISCUSSION TITLE
Define thumbnail URL from frontmatter?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [nunocoracao/blowfish#373] [Q&A] [answered]
Hi again 😄 

Is it possible to define the URL to the article thumbnail in the article frontmatter, rather than rely on the current [thumbnail instructions](https://blowfish.page/docs/configuration/#thumbnails)?

I suspect that this is not possible 😄 but figured I would still ask...

**Comment by nunocoracao** [comment, marked as answer]
Not at this stage.
